web/source/keyboards/kmwkeyboards.ts

```diff
@@ -383,6 +383,10 @@ namespace com.keyman.keyboards {
       // }
       this.doKeyboardChange(PInternalName, PLgCode);
 
+      p.catch(function() {
+        throw new Error("Unable to load keyboard with internal name \"" + PInternalName + "\", language code \"" + PLgCode + "\".");
+      })
```

Comment:
Can you add a comment on this? The message perhaps should be a little more concise as well (the term "internal name" is really pretty legacy these days 😉)

Reply:
... what sort of comment are you looking for?  "Shows up in Sentry error logs" or something?

Reply:
I was hoping for something that notes what can cause this to throw. When I read this, it took me a while to realise that `p` was a `Promise` and trying to understand when this might go wrong.